build-tools/automation/azure-pipelines.yaml

```
@@ -369,6 +369,12 @@ stages:
         testResultsFiles: TestResult-*.xml
         testRunTitle: Java Interop Tests - Windows Build Tree
 
+    - task: CmdLine@2
+      displayName: install apkdiff dotnet tool
+      inputs:
+        script: 'dotnet tool update apkdiff -g'
+      continueOnError: true
```

I think you could simply this one to the shorter `script` block:
```suggestion
    - script: dotnet tool update apkdiff -g
      displayName: install apkdiff dotnet tool
      continueOnError: true
```
It should be the same `.yaml` task, I think.